### ISSUE TITLE
Is it possible to access binary commands in Docker container?

### ISSUE THREAD

**Issue by stardiviner** [emacs-pe/docker-tramp.el#24]
I want to Emacs CIDER to use `clojure` binary in Docker container `clojure`.
So I taked a try with following config:
```emacs-lisp
(add-to-list 'exec-path "/docker:clojure:/usr/local/bin/")
(setq cider-clojure-cli-command "/docker:clojure:/usr/local/bin/clojure")
```
But it report error:
```
user-error: The /docker:clojure:/usr/local/bin/clojure executable isn’t on your ‘exec-path’
```

How can I do it?

**Comment by olessavluk** [comment]
In short - try and you will know. Because it depends from your use case :)

For local development - when I tried to run JS project with lot's of file watchers (on docker running on OS X host, over mapped filesystem) it was a lot slower than native. However on Linux the same setup works fine (due to native filesystem mapping). So for development on OS X I think it is easier to use everything native.

Also I hear that TRAMP [can be slow](https://stackoverflow.com/questions/1553093/emacs-tramp-performance) and [has poor support for multiple processes ](https://github.com/emacs-helm/helm/wiki/Grep#tramp)

But when I used it for doing sql - it was great, same as on my local computer but much more convenient.
